Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -482,6 +482,7 @@ You can set resource requests/limits for Rook components through the [Resource R
 It scrapes for Ceph daemon core dumps and sends them to the Ceph manager crash module so that core dumps are centralized and can be easily listed/accessed.
 You can read more about the [Ceph Crash module](https://docs.ceph.com/docs/master/mgr/crash/).
 * `logcollector`: Set resource requests/limits for the log collector. When enabled, this container runs as side-car to each Ceph daemons.
+* `rgwopslog`: Set resource requests/limits for the log collector. When enabled, this container runs as side-car to each Ceph daemons.
```

Comment:
Also, in documentation, make a note of what the sidecar is named to help users consume the feature.